Dockerfile

```
@@ -0,0 +1,16 @@
+FROM golang:1.11.2-alpine
+
+COPY . /go/src/github.com/rakyll/hey
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
+  go install -ldflags '-w -extldflags "-static"' \
+  /go/src/github.com/rakyll/hey
+
+FROM scratch
+
+COPY --from=0 /go/bin/hey /hey
```

```suggestion
COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
COPY --from=builder /go/bin/hey /hey
```

This way it also works when scheme is HTTPS.